### PR TITLE
[SMALLFIX] Improve documents in docs/cn/index.md

### PR DESCRIPTION
**Pull request by challvy**
Improve documents in docs/cn/index.md.

### CODE REVIEW

File: docs/cn/index.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+﻿---
```
Comment (yupeng9) [inline docs/cn/index.md]: restore this change?

Comment (challvy) [reply docs/cn/index.md]: How to do that？

Comment (yupeng9) [reply docs/cn/index.md]: This change is unnecessary. If you look at the button of 'show rich diff', it broke the table display.

Comment (challvy) [reply docs/cn/index.md]: OK. Now it's fixed.

**Pull request merged**